adapter/internal/discovery/xds/consul.go

```diff
@@ -35,14 +35,14 @@ const (
 	transportSocketName = "envoy.transport_sockets.tls"
 )
 
-func startConsulServiceDiscovery() {
-	for apiKey, clusterList := range openAPIClustersMap {
+func startConsulServiceDiscovery(organizationID string) {
```

Comment:
Where is this method called, We should call with org Id since the method signature has changed right?

Reply:
Yes. This is being called when the api is added. https://github.com/wso2/product-microgateway/pull/2042/files#diff-0bb37b8dc28a8ae4fbd9179a4b374745a9868b20fa545baa063548dfe63294b6R342